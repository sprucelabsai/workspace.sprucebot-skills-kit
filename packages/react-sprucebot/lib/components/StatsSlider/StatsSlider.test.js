'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _StatsSlider = require('./StatsSlider');

var _StatsSlider2 = _interopRequireDefault(_StatsSlider);

var _enzyme = require('enzyme');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('StatsSlider tests', function () {
	it('Should match the snapshot', function () {
		var renderedComponent = (0, _enzyme.shallow)(_react2.default.createElement(_StatsSlider2.default, null));

		expect(renderedComponent).toMatchSnapshot();
	});
});