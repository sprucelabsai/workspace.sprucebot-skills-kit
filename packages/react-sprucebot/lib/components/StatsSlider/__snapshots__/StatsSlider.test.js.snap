// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="stat__card__slider "
>
  <div
    className="stat__card__slider__scroll"
  />
</div>
`;
